src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -522,10 +522,20 @@ void ViewSceneDialog::pullCameraLookAt()
   auto spire = mSpire.lock();
   if(!spire) return;
 
-  auto lookAt = toDoubleVector(state_->getValue(Modules::Render::ViewScene::CameraLookAt).toVector());
+  auto lookAt = pointFromString(state_->getValue(Modules::Render::ViewScene::CameraLookAt).toString());
   spire->setCameraLookAt(glm::vec3(lookAt[0], lookAt[1], lookAt[2]));
 }
 
+//--------------------------------------------------------------------------------------------------
+glm::quat ViewSceneDialog::stringToQuat(std::string &s)
```

Comment:
Move this function to a new header/cc file, as a global function or in a new class, called ViewSceneUtility or the like.